miasm2/expression/expression_helper.py

```diff
@@ -533,3 +533,20 @@ def get(cls, size=32, depth=1, clean=True):
             cls.generated_elements = {}
 
         return got
+
+
+def expr_cmpu(arg1, arg2):
```

Comment:
Actually, you could rewrite it using `expr_cmps`, avoiding equation redundancy (and potential mistakes).
Indeed, `a <u b == (a <s b) ^ a.msb ^ b.msb`.